frontend/packages/kubevirt-plugin/integration-tests/tests/vm.dashboard.scenario.ts

```diff
@@ -0,0 +1,97 @@
+import { browser, ExpectedConditions as until } from 'protractor';
+import { testName } from '@console/internal-integration-tests/protractor.conf';
+import {
+  addLeakableResource,
+  createResource,
+  removeLeakedResources,
+} from '@console/shared/src/test-utils/utils';
+import { VirtualMachineModel } from '../../src/models';
+import {
+  vmDetailsName,
+  vmDetailsNamespace,
+  vmDetailsNode,
+  vmDetailsIPAddress,
+  vmStatus,
+  vmInventoryNICs,
+  vmInventoryDisks,
+} from '../views/dashboard.view';
+import { getVMManifest, hddDisk, multusNetworkInterface, multusNAD } from './utils/mocks';
+import { VirtualMachine } from './models/virtualMachine';
+import {
+  VM_STATUS,
+  VM_BOOTUP_TIMEOUT_SECS,
+  VM_ACTION,
+  VM_IMPORT_TIMEOUT_SECS,
+  TAB,
+  VM_STOP_TIMEOUT_SECS,
+  NOT_AVAILABLE,
+} from './utils/consts';
+
+describe('Test VM dashboard', () => {
+  const leakedResources = new Set<string>();
+  const testVM = getVMManifest('URL', testName, null, 'foo');
+
+  const vmName = `vm-dashboard-${testName}`;
+  let vm: VirtualMachine;
+
+  afterAll(async () => {
+    removeLeakedResources(leakedResources);
+  });
+
+  beforeAll(async () => {
+    createResource(multusNAD);
+    testVM.metadata.name = vmName;
+    createResource(testVM);
+    addLeakableResource(leakedResources, testVM);
+    vm = new VirtualMachine(testVM.metadata);
+    vm.navigateToDashboard();
+    await browser.wait(
+      until.textToBePresentInElement(vmStatus, VM_STATUS.Importing),
+      VM_IMPORT_TIMEOUT_SECS,
+    );
+    await browser.wait(
+      until.textToBePresentInElement(vmStatus, VM_STATUS.Off),
+      VM_IMPORT_TIMEOUT_SECS,
+    );
+  });
```

Comment:
Since we're waiting for the VM import it may be better to pass the timeout to this `beforeAll` as the default timeout for the whole spec is only 40 seconds.
It can be changed by passing 2nd parameter.
`beforeAll(functionBody, VM_IMPORT_TIMEOUT_SECS)`